content/test/integration/E2ETestEnvironment.ts

```diff
@@ -217,14 +223,18 @@ class PostgresWaitStrategy extends LogWaitStrategy {
   }
 }
 
+export function loadTestEnvironmentWithoutSynchronization(): E2ETestEnvironment {
```

Comment:
Let's make it simple by calling it something like `loadStandaloneTestEnvironment`